docker/postgres/Dockerfile

```
@@ -0,0 +1,21 @@
+# PostgreSQL 17 with Alpine Linux
+# Optimized for WV Wild Outdoors local development
+# Updated: December 2025
+
+FROM postgres:17-alpine
+
+# Install PostgreSQL extensions commonly needed
+RUN apk add --no-cache \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)